pkgs/top-level/haskell-packages.nix

```diff
@@ -91,7 +91,7 @@ in {
         # aarch64 ghc865Binary gets SEGVs due to haskell#15449 or similar
```

Comment:
You can drop the aarch64-darwin condition above, since stdenv.isAarch64 will also catch that.

Reply:
Yes, that makes sense.